R/seeding.R

```diff
@@ -0,0 +1,48 @@
+##' Seeding function for use with the particle filter
+##' 
+##' @title Compare model to ICU data
+##' 
+##' @param model An \code{odin_model} object
+##' 
+##' @param data The data, must be constructed with
+##'   \code{particle_filter_data}
+##'
+##' @param pars_seeding Parameters for seeding
+##' 
+##' @importFrom stats rpois
+##' 
+##' @export
+seeding_function <- function(model,
+                             data,
```

Comment:
Again, this would probably be better with the two parameters from data rather than the whole object

Reply:
Is this better? We use these four parameters from data https://github.com/mrc-ide/sircovid/blob/variable_seeding/R/seeding.R#L20-L23